io/shared/src/main/scala/fs2/io/file/Files.scala

```diff
@@ -573,6 +599,12 @@ object Files extends FilesCompanionPlatform {
             }
           }
     }
+
+    def writeUtf8(path: Path, flags: Flags): Pipe[F, String, Nothing] =
+      _.through(text.utf8.encode).through(writeAll(path, flags))
+
+    def writeUtf8Lines(path: Path, flags: Flags): Pipe[F, String, Nothing] =
+      _.through(text.lines).through(writeUtf8(path, flags))
```

Comment:
Out of curiosity can these implementations move into the parent trait, do they require any constraints?

Reply:
I must admit that I blindly copied the method used for readUtf8 and its sibling. Also, WDYM with constraint here?

Reply:
> do they require any constraints?
I got it, the `Async`. And no, they don't require it and neither do readUtf8, readUtf8Lines and readCursor apparently. I can move the newly added methods to the trait, what about readUtf8 and readUtf8Lines?

Reply:
Great, thanks! Sure, I say move them up.

Reply:
Btw, I'm not sure this does what we want. Doesn't `text.lines` split a stream of `String` into lines, not the other way around? e.g. add `\n` between `String`s?

Might be good to add a test 😇

Reply:
You're probably right, I'll add a few